### PR TITLE
Polkadot v0.9.37

### LINKED ISSUE #301: Bump to polkadot-v0.9.37
The time has come, where we should do an upgrade with the node again, before we lag too far behind.

1. Bump substratedependencies to polkadot-v0.9.37
2. Change our dependencies to the polkadot-v0.9.37 branch in the palles, or just patch them with the local deps you have.
3. If everything compiles fine, otherwise check the diff of [node](https://github.com/encointer/encointer-node/tree/master/node) to [substrate/bin/node-template/node](https://github.com/paritytech/substrate/tree/master/bin/node-template/node).
4. If the runtime does not compile, we can probably just handle the compiler errors one by one.

### PR DESCRIPTION
**Pull request by pifragile**
closes #301

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -4682,7 +4984,7 @@ dependencies = [
 [[package]]
 name = "pallet-encointer-scheduler"
 version = "1.1.0"
-source = "git+https://github.com/encointer/pallets?branch=master#f859dff21f9c9fd3863f6eaa5cf64cd1f827c7af"
+source = "git+https://github.com/encointer/pallets?branch=master#61e42c00aa5b2e2759d718b54abf4bb6a4d3199f"
```
Comment (clangenb) [inline Cargo.lock]: We seem to be inconsistent here, can you make sure that these deps also point to the polkdot-v0.9.37 branch?

**Review by clangenb — COMMENTED** [review]
Beautiful, one tiny detail and we are done.

**Comment by clangenb** [comment]
This commit fixed the srtool build in the integritee-node: https://github.com/integritee-network/integritee-node/pull/159/commits/fa586e5420bbee3f7ed12e58bf11b0d028e6467f

File: Cargo.lock (hunk 1 above)
Comment (pifragile) [reply Cargo.lock]: oh ok, not to master? i still dont understand our system with those version branches

Comment (clangenb) [reply Cargo.lock]: Same as the other encointer-pallet dependencies: polkadot-v0.9.37.

I think the system did change over the time. I want us to just depend on the polkadot-vx.x.x branches of our pallets. That this was different, was just an error.

**Review by clangenb — APPROVED** [review]

**Pull request merged**